src/go/rpk/pkg/cli/cmd/redpanda/config.go

```diff
@@ -147,6 +147,12 @@ func bootstrap(fs afero.Fs) *cobra.Command {
 			cfg.Redpanda.SeedServers = []config.SeedServer{}
 			cfg.Redpanda.SeedServers = seeds
 
+			// The Redpanda binary defaults to the root-driven style of cluster
+			// formation. In bootstrapping a new cluster, use seeds-driven
+			// cluster formation.
+			cfg.Redpanda.Other = make(map[string]interface{})
```

Comment:
Other might be non-nil here: this should only initialize the map if it is nil

Reply:
Done